server/src/main/java/io/unitycatalog/server/service/AuthService.java

```diff
@@ -167,17 +168,37 @@ public HttpResponse grantToken(
             String cookieTimeout =
                 ServerProperties.getInstance().getProperty("server.cookie-timeout", "P5D");
             Cookie cookie =
-                Cookie.secureBuilder(AuthDecorator.UC_TOKEN_KEY, accessToken)
-                    .path("/")
-                    .maxAge(Duration.parse(cookieTimeout).getSeconds())
-                    .build();
+                createCookie(
+                    AuthDecorator.UC_TOKEN_KEY,
+                    accessToken,
+                    "/",
```

Comment:
Can we give this a name?

Reply:
hi @jaceklaskowski  couldnt get you so  which part of the code you want to give name? is it the path parameter?

Reply:
+1 what are you suggesting to "name"?